src/lib/layout/index.spec.ts

```diff
@@ -30,6 +30,9 @@ rootLogger.settings({
   },
 })
 
+// Force stdout width to not wrap the logs and mess with the snapshots
+process.stdout.columns = 300
```

Comment:
Don't think this is needed afterall

Reply:
It is, the logger reacts to columns and the snapshot were failing without it, even after the `fatal` refactor